kotlin-inject-compiler/core/src/main/kotlin/me/tatarka/inject/compiler/InjectGenerator.kt

```diff
@@ -30,6 +30,7 @@ val INJECT = ClassName(ANNOTATION_PACKAGE_NAME, "Inject")
 val INTO_MAP = ClassName(ANNOTATION_PACKAGE_NAME, "IntoMap")
 val INTO_SET = ClassName(ANNOTATION_PACKAGE_NAME, "IntoSet")
 val ASSISTED = ClassName(ANNOTATION_PACKAGE_NAME, "Assisted")
+val TARGET_COMPONENT_ACCESSOR = ClassName(ANNOTATION_PACKAGE_NAME, "TargetComponentAccessor")
```

Comment:
is this used?

Reply:
I was experimenting with a few things. I'll take it out if it doesn't end up getting used.